frontend/src/services/PokemonNormalizer.ts

```diff
@@ -0,0 +1,10 @@
+import { PokemonInterface } from 'components/Pokemon/Pokemon';
+import { PokemonState } from 'redux/Pokemon';
+
+const normalize = (array: PokemonInterface[]) =>
+  array.reduce(
+    (object: PokemonState, item: PokemonInterface) => ({ ...object, [item.id]: item }),
+    {},
+  );
+
+export { normalize };
```

Comment:
Just a remark, the normalization here is simple, when the become harder with nested object, we can use Normalizr https://github.com/paularmstrong/normalizr